### DISCUSSION TITLE
Is it possible to make the section "Mail Settings" translatable?

### DISCUSSION THREAD

**Discussion by devmuaz** [pocketbase/pocketbase#4584] [Q&A]
I've been using Pocketbase for a while now, and it makes things easier to work with, but I was wondering in case my app (mobile app that uses the Pocketbase as a backend) has multiple languages and wanted to send emails (auth emails) based on the user's language preferences?

Currently, I'm sending the emails in English.

Is it doable?

**Comment by ganigeorgiev** [comment]
There is no builtin mechanism for handling email translations at the moment (see https://github.com/pocketbase/pocketbase/issues/593).
